model/src/main/proto/exploration_checkpoint.proto

```diff
@@ -46,6 +46,9 @@ message ExplorationCheckpoint {
 
   // The timestamp in milliseconds of when the checkpoint was saved for the first time.
   int64 timestamp_of_first_checkpoint = 9;
+
+  // The saved help index for the exploration.
```

Comment:
```suggestion
  // The saved help index for the exploration indicating the revealed & available hints/solution of the latest state.
```

Reply:
Also, are 'solution_is_revealed' & 'hint_index' still needed now that this is here?